standard/template/sections/clause_4_terms_and_definitions.adoc

```diff
@@ -1,17 +1,64 @@
-== Terms and definitions
+== Terms, definitions and abbreviated terms
 
-This document uses the terms defined in Sub-clause 5.3 of <<OGC06-121r9>>, which is based on the ISO/IEC Directives, Part 2, Rules for the structure and drafting of International Standards. In particular, the word "`shall`" (not "`must`") is the verb form used to indicate a requirement to be strictly followed to conform to this standard.
+=== Terms and definitions
 
-For the purposes of this document, the following additional terms and definitions apply.
+==== array
 
-=== example term
+A multidimensional, regularly spaced collection of values (e.g., raster data or gridded measurements), typically indexed by dimensions such as time, latitude, longitude, or spectral band.
 
-term used for exemplary purposes
+==== chunk
 
-[.source]
-<<ISO19101-1>>
+A sub-array representing a partition of a larger array, used to optimise data access and storage. In Zarr, data is stored and accessed as a collection of independently compressed chunks.
 
-NOTE: An example note.
+==== coordinate variable
 
-[example]
-Here's an example of an example term.
+A one-dimensional array whose values define the coordinate system for a dimension of one or more data variables. Typical examples include latitude, longitude, time, or vertical levels.
```

Comment:
Would it be useful to link to CDM / CF conventions and / or state whether this definition is identical to those other definitions?

Reply:
I agree.